app/pages/collections/list.cjsx

```diff
@@ -65,6 +65,9 @@ List = React.createClass
       .then (collections) =>
         @setState {collections}
 
+  shared: (collection) ->
```

Comment:
I don't think it should be compared to the logged in user since the `collections/:collection_owner` route with any public collections can be viewed while logged out. Logged out users should be able to see if the collection is owned by the `:collection_owner` user or is a collab collection and who the owner is.